tools/depends/target/cmakebuildsys/Makefile

```diff
@@ -0,0 +1,16 @@
+-include ../../Makefile.include
+
+VERSION.TXT := $(XBMCROOT)/version.txt
+APP_NAME=$(shell awk '/APP_NAME/ {print tolower($$2)}' $(VERSION.TXT))
+
+all: $(XBMCROOT)/lib$(APP_NAME).so
+
+$(XBMCROOT)/lib$(APP_NAME).so:
```

Comment:
phony would be ok for now, since cmake will take to not needlessly rebuild anyway.